mypy/util.py

```diff
@@ -69,10 +69,14 @@ def try_find_python2_interpreter() -> Optional[str]:
         return _python2_interpreter
     for interpreter in default_python2_interpreter:
         try:
-            process = subprocess.Popen([interpreter, '-V'], stdout=subprocess.PIPE,
-                                       stderr=subprocess.STDOUT)
-            stdout, stderr = process.communicate()
-            if b'Python 2.7' in stdout:
+            retcode = subprocess.Popen(
+                [
+                    interpreter,
+                    '-c',
+                    'import sys, typing; assert sys.version_info[:2] == (2, 7)'
+                ],
+            ).wait()
```

Comment:
Please don't use this incredibly stretched vertical layout; it hurts my eyes and does not help me understand the code.